src/github/graphql.ts

```diff
@@ -0,0 +1,175 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
Love this file. It would be *great* if we could autogenerate it from our queries, though. Maybe there's a loader that'll do it for us?

Reply:
I think technically we can https://medium.com/graphql-mastery/generate-javascript-static-types-from-graphql-typescript-and-flow-4d28b46b8d13